collect_app/src/androidTest/java/org/odk/collect/android/support/pages/NotificationDrawer.kt

```diff
@@ -29,7 +30,7 @@ class NotificationDrawer {
         appName: String,
         title: String,
         body: String? = null,
-        subtext: String? = null
+        subtext: String = Project.DEMO_PROJECT_NAME
```

Comment:
I think I'd prefer this to be explicitly in the tests rather than a default. It seems strange for `NotificationDrawer` to know or care about `Project`.